playground/backend/internal/fs_tool/fs.go

```diff
@@ -171,3 +172,10 @@ func (l *LifeCycle) GetAbsoluteBaseFolderPath() string {
 	absoluteFilePath, _ := filepath.Abs(l.Folder.BaseFolder)
 	return absoluteFilePath
 }
+
+// GetAbsoluteLogFilePath returns absolute path to the logs file (/path/to/workingDir/executable_files/{pipelineId}/logs.log)
+func (l *LifeCycle) GetAbsoluteLogFilePath() string {
+	filePath := filepath.Join(l.Folder.BaseFolder, logFileName)
+	absoluteFilePath, _ := filepath.Abs(filePath)
```

Comment:
Not sure that we need to process errors here, but let's double-check which cases we can take an error?

Reply:
`filepath.Abs`could return an error in case of `os.Getwd()` returns an error:
```
if IsAbs(path) {
	return Clean(path), nil
}
wd, err := os.Getwd()
if err != nil {
	return "", err
}
return Join(wd, path), nil
```